azure-pipelines/signing.yml

```
@@ -307,6 +307,43 @@ extends:
               mv "$(Agent.TempDirectory)/build/vcpkg" "$(Build.ArtifactStagingDirectory)/vcpkg-muslc"
+      - job: glibc_arm64_build
+        displayName: 'glibc Arm64 Build'
+        dependsOn:
+        - arch_independent
+        pool:
+          name: 'vcpkg-mariner-aarch64-1espt'
+          os: linux
+        variables:
+          VCPKG_STANDALONE_BUNDLE_SHA: $[ dependencies.arch_independent.outputs['shas.VCPKG_STANDALONE_BUNDLE_SHA'] ]
+          VCPKG_BASE_VERSION: $[ dependencies.arch_independent.outputs['versions.VCPKG_BASE_VERSION'] ]
+        templateContext:
+          sdl:
+            codeql:
+              language: cpp
+              buildIdentifier: vcpkg_cpp_glibc_arm64
+          mb:
+            signing:
+              enabled: false
+          outputs:
+          - output: pipelineArtifact
+            displayName: 'Publish Unsigned glibc Arm64 Binary'
+            artifactName: 'stagingGlibcArm64'
+            publishLocation: 'Container'
+            targetPath: '$(Build.ArtifactStagingDirectory)'
+        steps:
+          - bash: |
+              az login --identity --username 29a4d3e7-c7d5-41c7-b5a0-fee8cf466371
+              az acr login --name vcpkgdockercontainers
+            displayName: 'Set up managed identity'
+          - task: CmdLine@2
+            displayName: "Run Docker build for arm64 Linux binary"
+            inputs:
+              failOnStderr: false
+              script: |
+                mkdir -p "$(Agent.TempDirectory)/build"
+                docker run --rm --mount "type=bind,source=$(Build.Repository.LocalPath),target=/source,readonly" --mount "type=bind,source=$(Agent.TempDirectory)/build,target=/build" vcpkgdockercontainers.azurecr.io/vcpkg/vcpkg-arm64-linux:2025-02-10 sh -c "cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DVCPKG_DEVELOPMENT_WARNINGS=ON -DVCPKG_WARNINGS_AS_ERRORS=ON -DVCPKG_BUILD_FUZZING=OFF -DVCPKG_EMBED_GIT_SHA=ON -DVCPKG_OFFICIAL_BUILD=ON -DCMAKE_CXX_FLAGS=\"-static -s -static-libgcc -static-libstdc++\" -DVCPKG_CMAKERC_URL=$(cmakerc-tarball-url) -DVCPKG_FMT_URL=$(fmt-tarball-url) -DVCPKG_STANDALONE_BUNDLE_SHA=$(VCPKG_STANDALONE_BUNDLE_SHA) -DVCPKG_BASE_VERSION=$(VCPKG_BASE_VERSION) -DVCPKG_VERSION=$(Build.SourceVersion) -S /source -B /build 2>&1 && ninja -C /build"
```

```suggestion
                docker run --rm --mount "type=bind,source=$(Build.Repository.LocalPath),target=/source,readonly" --mount "type=bind,source=$(Agent.TempDirectory)/build,target=/build" vcpkgdockercontainers.azurecr.io/vcpkg/vcpkg-arm64-linux:2025-02-10 sh -c "cmake -G Ninja -DCMAKE_TOOLCHAIN_FILE=/source/azure-pipelines/vcpkg-arm64/toolchain.cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DVCPKG_DEVELOPMENT_WARNINGS=ON -DVCPKG_WARNINGS_AS_ERRORS=ON -DVCPKG_BUILD_FUZZING=OFF -DVCPKG_EMBED_GIT_SHA=ON -DVCPKG_OFFICIAL_BUILD=ON -DVCPKG_CMAKERC_URL=$(cmakerc-tarball-url) -DVCPKG_FMT_URL=$(fmt-tarball-url) -DVCPKG_STANDALONE_BUNDLE_SHA=$(VCPKG_STANDALONE_BUNDLE_SHA) -DVCPKG_BASE_VERSION=$(VCPKG_BASE_VERSION) -DVCPKG_VERSION=$(Build.SourceVersion) -S /source -B /build 2>&1 && ninja -C /build"
```

It might be clearer to reorder this whole inserted block before alpine (so the glibc builds are together)